tdtax/binary.yaml

```diff
@@ -0,0 +1,37 @@
+class: Binary
```

Comment:
I'd rename this file `eclipsing.yaml` and remove `Binary` as the top-level class. Also remove `Ellipsoidal` (which I think should is better as a tag)

Reply:
Should I leave the rest of the eclipsing.yaml subclasses the same (ie still include all the binary classes and use eclipsing as a tag for a subset of them)? Or should there be a separate binary yaml file?

Reply:
I guess you can move the `FS CMa` to the eruptive sublcass and keep all the others here under eclipsing. Just do `git mv binary.yaml eclipsing.yaml` to properly rename this file.